packages/fast-components/src/styles/display.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from "@microsoft/fast-element";
+
+export type CSSDisplayPropertyValue =
+    | "inline"
+    | "block"
+    | "contents"
+    | "flex"
+    | "grid"
+    | "inline-block"
+    | "inline-flex"
+    | "inline-grid"
+    | "inline-table"
+    | "list-item"
+    | "run-in"
+    | "table"
+    | "table-caption"
+    | "table-column-group"
+    | "table-header-group"
+    | "table-footer-group"
+    | "table-row-group"
+    | "table-cell"
+    | "table-column"
+    | "table-row"
+    | "none"
+    | "initial"
+    | "inherit";
+
+const hidden = css`
```

Comment:
I like that, I think we should have some kind of documented write up though so contributors can know why we decide to use different methods for creating css.

Reply:
My thought here was that *every* component should have this, so associating this sheet with every component rather than every component enumerating the same CSS would be more efficient. If that isn't the case I'm happy to just make it a string - just thought the re-use of the adopted stylesheet would be beneficial

Reply:
Changed to be a string type